src/NuGet.Core/NuGet.CommandLine.XPlat/NuGet.CommandLine.XPlat.csproj

```diff
@@ -12,6 +12,7 @@
 
   <ItemGroup>
     <ProjectReference Include="..\NuGet.Commands\NuGet.Commands.csproj" />
+    <ProjectReference Include="..\NuGet.PackageManagement\NuGet.PackageManagement.csproj" />
```

Comment:
NuGet.PackageManagement is largely packages.config, which explicitly isn't supported by the .NET SDK. NuGet.PackageManagement also contains abstractions used by our VS code, point being that those classes also should not be used in the dotnet cli.  In any case, I don't see any `using NuGet.PackageManagement` statements, so I belive that this project reference is not needed.